systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaResource.java

```diff
@@ -55,7 +55,7 @@ public static DoneableKafka kafkaEphemeral(String name, int kafkaReplicas, int z
     }
 
     public static DoneableKafka kafkaPersistent(String name, int kafkaReplicas) {
-        return kafkaPersistent(name, kafkaReplicas, 3);
+        return kafkaPersistent(name, kafkaReplicas, Math.min(kafkaReplicas, 3));
```

Comment:
I think I getting your point by these. You are using the `kafkaReplicas` parameter that is dependent on the zookeeperReplicas. Wouldn't be better using some `Math.min(kafkaReplicas / 3, 3)` meaning if we specify 3 Kafka replicas we will deploy 1 zookeeper instead of 3, which are not needed? WDYT?

Reply:
Well in that case we will need at least 9 brokers to have 3 zk. It doesn't sound me like a good idea. You always can specify zk replicas by third param.